bin/methods/start.js

```diff
@@ -26,14 +30,18 @@ function start (flags, plugins = []) {
   })
 
   // Set defaults
-  if (!config.loader) config.loader = path.join(__dirname, '..', 'loader')
+  if (!config.loader) config.loader = path.join(__dirname, '../../node_modules/cordlr-loader')
```

Comment:
can this not just be `cordlr-loader` because node resolves npm package names automatically?